src/Modules/CalcOffence.lua

```diff
@@ -2677,27 +2677,36 @@ function calcs.offence(env, actor, activeSkill)
 		else
 			output.KnockbackChanceOnHit = skillModList:Sum("BASE", cfg, "EnemyKnockbackChance")
 		end
-		if skillModList:Sum("BASE", cfg, "ScorchChance") > 0 then
+		if skillModList:Sum("BASE", cfg, "EnemyScorchChance") > 0 then
```

Comment:
Is there a reason you didn't also roll these up into a loop?  If you do, would be nice to add support for `CritsDontAlwaysFreeze` for the other ailments.  Mod appears on Painseeker that enables that.